src/pyOpenMS/pxds/BayesianProteinInferenceAlgorithm.pxd

```diff
@@ -93,12 +91,11 @@ cdef extern from "<OpenMS/ANALYSIS/ID/BayesianProteinInferenceAlgorithm.h>" name
         void inferPosteriorProbabilities(ConsensusMap & cmap,
                                          bool greedy_group_resolution) nogil except +
           # wrap-doc:
-          #   Performs inference
-          #   -----
           #   Writes its results into protein and (optionally also) peptide hits (as new score)
```

Comment:
Since lines seem to be concatenated without newlines in Sphinx, I think we need to add "." (colons) at the end of every line if a sentence ends there.